### PR TITLE
CodeCore Blog Assignment

### PR DESCRIPTION
**Pull request by parkerdan**

### CODE REVIEW

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Place all the styles related to the users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+.sign-up-form {
```
Comment (parkerdan) [inline app/assets/stylesheets/users.scss]: could refactor and combine with top file

**Comment by EricaNichol** [comment]
Dan , nice job done. more cats next time.

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  def create
+    @comment = Comment.new comment_params
+    @post = Post.find params[:post_id]
+    @comment.post = @post
+    @comment.user = current_user
+    respond_to do |format|
+      if @comment.save
+        #  CommentsMailer.notify_post_owner(@comment).deliver_later
+
```
Comment (EricaNichol) [inline app/controllers/comments_controller.rb]: More Cats

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def formatted_date(date)
+      date.strftime("%Y-%b-%d")
+    end
+
```
Comment (EricaNichol) [inline app/helpers/application_helper.rb]: Maybe typing out what X is suppose to be.

File: app/views/comments/_form.html.erb [hunk 4]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (EricaNichol) [inline app/views/comments/_form.html.erb]: Tam mentioned that leaving the form in the modal might be the best practice. Something to look in to would be putting the form in a partial for AJAX later down the road.

**Pull request merged**